### PR TITLE
(test):Common utils test coverage

### PR DESCRIPTION
**Pull request by gitcliff**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
Adding test coverage for some common util functions

### CODE REVIEW

File: src/utils/common-utils.test.ts [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+}));
+
+jest.mock('../validators/form-validator', () => ({
+  isEmpty: jest.fn(),
+}));
+
+describe('utils functions', () => {
+  describe('flattenObsList', () => {
+    it('should flatten a nested obs list', () => {
+      const obsList: OpenmrsObs[] = [
```
Comment (arodidev) [inline src/utils/common-utils.test.ts:17]: We should consider moving this to the mocks directory just to have a cleaner code base.

**Review by arodidev — COMMENTED** [review]
Thanks for the work @gitcliff, left a minor comment.

File: src/utils/common-utils.test.ts (hunk 1 above)
Comment (gitcliff) [reply src/utils/common-utils.test.ts]: thanks @arodidev 
Done

**Review by arodidev — APPROVED** [review]
LGTM @gitcliff

**Pull request merged**